lively.collab/comments/commentIndicator.js

```diff
@@ -111,6 +102,27 @@ export class CommentIndicator extends Label {
     this.position = morphOrigin.addPt(pt(xOffset, yOffset));
   }
 
+  /*
+  DELETION, COPYING
+  */
+  abandon () {
+    const referenceMorph = this.referenceMorph;
+    disconnect(referenceMorph, 'onChange', this, 'referenceMoving');
+    disconnect(referenceMorph, 'onOwnerChanged', this, 'connectMorphs');
+    super.abandon();
+  }
+
+  get isCommentIndicator () {
+    return true;
```

Comment:
why is this at deletion copying? I know it is used in this context, but I don't think it fits here

Reply:
+1
Would put it just below the constructor, without any specific category